### ISSUE TITLE
False positive for RSpec/PendingWithoutReason in 2.16.0 when object has a method pending

### ISSUE THREAD

**Issue by sihu** [rubocop/rubocop-rspec#1549]
Hi rubocop-rspec team,

I noticed that we had a false positive in our project (at least i think it's a false positive): There is a object with an enum `state` with the value `pending`. In our test we are querying all pending transations. I would expect that the spec does not consider this as an issue.

```
    it 'approves the transaction' do
      expect do
        request
      end.to change(Transaction.pending, :count).from(1).to(0)
```

(Ruby 2.7.7, rubocop 1.43.0, rubocop-ast 1.24.1, rubocop-rspec 2.16.0)

**Comment by ydah** [comment]
Thank you for your report!
This issue has been fixed in https://github.com/rubocop/rubocop-rspec/pull/1515 and https://github.com/rubocop/rubocop-rspec/pull/1516, but not yet released. Please wait for the next release.

**Comment by pirj** [comment]
My apologies for a temporary inconvenience.

@bquorning does it make sense to release bug fixes in a patch release before 2.17 with Capybara extraction?
